vagrant/config/vagrant-local.example.yml

```diff
@@ -12,8 +12,11 @@ box_check_update: false
 # Virtual machine name
 machine_name: y2aa
 
+# Virtual netmask
+netmask: 192.168.83
+
 # Virtual machine IP
-ip: 192.168.83.137
+ip: "137"
```

Comment:
better be explicit here, its confusing to use existing terms like `netmask` and `ip` and put values in there that do not match the expectation.
If you have multiple machines, allow specifying their ips by "web_ip", "mysql_ip", ...